upcloud/resource_upcloud_server.go

```diff
@@ -740,3 +759,11 @@ func resourceUpCloudServerDelete(ctx context.Context, d *schema.ResourceData, me
 
 	return diags
 }
+
+func cloudServerDefaultTitleFromHostname(hostname string) string {
+	suffix := " (managed by terraform)"
+	if len(hostname)+len(suffix) > cloudServerTitleLength {
+		hostname = fmt.Sprintf("%s...", hostname[:cloudServerTitleLength-len(suffix)-3])
```

Comment:
UpCloud API accepts the single character ellipsis `…`
Could save two characters by using that here 😉